cl/simple_pages/templates/components.html

```diff
@@ -6,6 +6,8 @@
 {% tailwind_css %}
 {% tailwind_preload_css %}
 
+<script src="{% static 'js/alpinejscsp@3.14.8.min.js' %}" defer init nonce="{{ request.csp_nonce }}"></script>
```

Comment:
Hm, do we want this here or in new_base.html? I guess if we put it here we can just remember to include it only on the pages that need it; maybe that's better if we don't need it on a lot of pages.